vhdbuilder/packer/install-dependencies.sh

```diff
@@ -464,8 +464,9 @@ MULTI_ARCH_KUBE_BINARY_VERSIONS="
 1.21.9-hotfix.20220204
 1.22.4-hotfix.20220201
 1.22.6-hotfix.20220130
-1.23.3-hotfix.20220130
-1.23.4
+1.23.3-hotfix.20220401.1
```

Comment:
will this require a change for the post VHD tests as well?

Reply:
yes, it will.

Reply:
i meant this file right here https://github.com/Azure/AgentBaker/blob/master/vhdbuilder/packer/test/linux-vhd-content-test.sh

Reply:
nice catch, thanks! I've updated the test file.